### PR TITLE
Chart : resources by default

### PR DESCRIPTION
**Pull request by olevitt**
Exact values are up for discussion

### CODE REVIEW

**Comment by garronej** [comment]
Hey @olevitt,  
It seems both a little high for the API and low for the WEB but really I have no idea.  
Onyxia web, despide being an SPA serves a lot of little files beacause the APP is split into multiple little files that are downloaded lazily.  I'm surprised that Ngnix can conforably serves all that with so few resources but really I have no real idea, you are much more experienced with production that I am so I trust your jugement on this (and I'm too lazy to mesure).  

If @fcomte is ok let's merge

**Comment by olevitt** [comment]
@garronej this PR is now ok to merge.  
We should add a note to the changelog indicating that the Chart now has default resources requests that may or may not suit users needs. How to do that ?

**Comment by odysseu** [comment]
> @garronej this PR is now ok to merge. We should add a note to the changelog indicating that the Chart now has default resources requests that may or may not suit users needs. How to do that ?

- For end-users it could be to add a banner : https://github.com/InseeFrLab/onyxia/blob/main/web/.env#L583-L615
Even though any end-user will see the resources's sliders' values. 👀 

- For other onyxia implementers, just a changelog message ?

**Comment by garronej** [comment]
> We should add a note to the changelog indicating that the Chart now has default resources requests that may or may not suit users needs. How to do that ?  

I take care of it!

**Comment by garronej** [comment]
> For end-users it could be to add a banner : https://github.com/InseeFrLab/onyxia/blob/main/web/.env#L583-L615
Even though any end-user will see the resources's sliders' values. 👀  

I think it's more for the instance administrator that for the end user. I guess a notice in the migration guide to v7 should be enough.

File: helm-chart/Chart.yaml [hunk 1]
```diff
@@ -14,4 +14,4 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 6.5.1
+version: 6.6.0
```
Comment (garronej) [inline helm-chart/Chart.yaml]:
```suggestion
version: 6.5.1
```
We'll release this with v7

**Pull request merged**